minio/scripts/file-operations.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# Executable that waits for a port at an address to be open.
+# For further details go to: https://github.com/ufoscout/docker-compose-wait
+/wait
+
+echo "Add s3 location alias called storage to minio config"
+mc config host add storage http://minio:9000 longkey verysecretkey
+
+echo "Create private bucket (private by default)"
```

Comment:
misses the similar "Create public bucket (files are read-only, no listings)"

Reply:
should be there now.